diff --git a/packages/scratch-gui/src/lib/libraries/extensions/index.jsx b/packages/scratch-gui/src/lib/libraries/extensions/index.jsx
index 276611ddc..2a436496c 100644
--- a/packages/scratch-gui/src/lib/libraries/extensions/index.jsx
+++ b/packages/scratch-gui/src/lib/libraries/extensions/index.jsx
@@ -1,6 +1,9 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import yourExtensionInsetIconURL from './yourextension/your-extension-icon.png';
+import yourExtensionIconURL from './yourextension/your-extension-background.png';
+
 import musicIconURL from './music/music.png';
 import musicInsetIconURL from './music/music-small.svg';
 
@@ -50,6 +53,14 @@ import faceSensingIconURL from './faceSensing/faceSensing.png';
 import faceSensingInsetIconURL from './faceSensing/faceSensing-small.svg';
 
 export default [
+    {
+        name: 'AttaBotSTEM',
+        extensionId: 'yourScratchExtension',
+        iconURL: yourExtensionIconURL,
+        insetIconURL: yourExtensionInsetIconURL,
+        description: 'Extensión de bloques para controlar el robot AttaBot',
+        featured: true
+    },
     {
         name: (
             <FormattedMessage
