diff --git a/src/lib/libraries/extensions/index.jsx b/src/lib/libraries/extensions/index.jsx
index ba18b91..e316f38 100644
--- a/src/lib/libraries/extensions/index.jsx
+++ b/src/lib/libraries/extensions/index.jsx
@@ -1,6 +1,9 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import yourExtensionInsetIconURL from './yourextension/your-extension-icon.png';
+import yourExtensionIconURL from './yourextension/your-extension-background.png';
+
 import musicIconURL from './music/music.png';
 import musicInsetIconURL from './music/music-small.svg';
 
@@ -47,6 +50,15 @@ import gdxforConnectionIconURL from './gdxfor/gdxfor-illustration.svg';
 import gdxforConnectionSmallIconURL from './gdxfor/gdxfor-small.svg';
 
 export default [
+    {
+        name: 'AttaScratch',
+        extensionId: 'AttaBotSTEM',
+        iconURL: yourExtensionIconURL,
+        insetIconURL: yourExtensionInsetIconURL,
+        description: 'Extensión para el AttaBot-STEM',
+        featured: true,
+        disabled: false
+    },
     {
         name: (
             <FormattedMessage
diff --git a/src/lib/default-project/index.js b/src/lib/default-project/index.js
index a1b2c3d..e4f5g6h 100644
--- a/src/lib/default-project/index.js
+++ b/src/lib/default-project/index.js
@@ -1,52 +1,14 @@
-import projectData from './project-data';
-  
 /* eslint-disable import/no-unresolved */
-import popWav from '!arraybuffer-loader!./83a9787d4cb6f3b7632b4ddfebf74367.wav?';
-import meowWav from '!arraybuffer-loader!./83c36d806dc92327b9e7049a565c6bff.wav?';
-import backdrop from '!raw-loader!./cd21514d0531fdffb22204e0ec5ed84a.svg?';
-import costume1 from '!raw-loader!./bcf454acf82e4504149f7ffe07081dbc.svg?';
-import costume2 from '!raw-loader!./0fb9be3e8397c983338cb71dc84d0b25.svg?';
+import plantillaData from '!arraybuffer-loader!./yourextension/plantilla.sb3';
 /* eslint-enable import/no-unresolved */
 
-const defaultProject = translator => {
-    let _TextEncoder;
-    if (typeof TextEncoder === 'undefined') {
-        _TextEncoder = require('fastestsmallesttextencoderdecoder').TextEncoder;
-    } else {
-        _TextEncoder = TextEncoder;
-    }
-    const encoder = new _TextEncoder();
- 
-    const projectJson = projectData(translator);
-    return [{
-        id: 0,
-        assetType: 'Project',
-        dataFormat: 'JSON',
-        data: JSON.stringify(projectJson)
-    }, {
-        id: '83a9787d4cb6f3b7632b4ddfebf74367',
-        assetType: 'Sound',
-        dataFormat: 'WAV',
-        data: new Uint8Array(popWav)
-    }, {
-        id: '83c36d806dc92327b9e7049a565c6bff',
-        assetType: 'Sound',
-        dataFormat: 'WAV',
-        data: new Uint8Array(meowWav)
-    }, {
-        id: 'cd21514d0531fdffb22204e0ec5ed84a',
-        assetType: 'ImageVector',
-        dataFormat: 'SVG',
-        data: encoder.encode(backdrop)
-    }, {
-        id: 'bcf454acf82e4504149f7ffe07081dbc',
-        assetType: 'ImageVector',
-        dataFormat: 'SVG',
-        data: encoder.encode(costume1)
-    }, {
-        id: '0fb9be3e8397c983338cb71dc84d0b25',
-        assetType: 'ImageVector',
-        dataFormat: 'SVG',
-        data: encoder.encode(costume2)
-    }];
-};
+// eslint-disable-next-line no-unused-vars
+const defaultProject = _translator => [{
+    id: 0,
+    assetType: 'Project',
+    dataFormat: 'JSON',
+    data: new Uint8Array(plantillaData)
+}];
 
 export default defaultProject;